openedx_events/event_bus/avro/tests/test_utilities.py

```diff
@@ -1,19 +1,22 @@
 """
-Utility methods and classes for testing AvroAttrsBridge
+Utility methods and classes for testing AvroSchemaGenerator.
```

Comment:
Unless you want to be more specific:
```suggestion
Utility methods and classes for testing with Avro.
```